src/lib/Components/Browse.php

```diff
@@ -21,8 +21,6 @@ protected function getLocationId()
         if ($this->request->attributes->get('location')) {
             return $this->request->attributes->get('location')->id;
         }
-
-        return 'false';
```

Comment:
that's a bit short, this method should always return *something* and the `_toString` and `jsonSerialize` should be updated accordingly